java/templates/service.yaml

```diff
@@ -5,6 +5,15 @@ metadata:
   name: {{ template "hmcts.java.releaseName" . }}
   labels:
     {{- ( include "java.labels" . ) | indent 4 }}
+  {{- /*
+  # WARNING: idamIngressSessionAffinity is a temporary option only approved for SIDAM
+  # Do not use without approval from RPE
+  */ -}}
+  {{- if (.Values.idamIngressSessionAffinity.enabled) }}
```

Comment:
This was a conscious decision just to help indicate that it should not be used by others.

Reply:
I would just add ability to add annotations as a list then ? and mention nothing about session affinity.

Reply:
something like https://github.com/helm/charts/blob/e5c7915aa958bc38236145db19ef60d23409f292/stable/prometheus-operator/templates/prometheus-operator/deployment.yaml#L24-L26

Reply:
i'm not sure if we want that, it's nice to have control over what goes in there, the idam bit is really just to stop others using it, it could be dropped and cookie name made configurable but we just really don't want people using this.

I asked henry not to document it as well

Reply:
I really don't like having idam name in chart-java , but believe in democracy :)